lib/operations/mongo_client_ops.js

```diff
@@ -413,7 +413,7 @@ function createTopology(mongoClient, topologyType, options, callback) {
   // Open the connection
   topology.connect(options, (err, newTopology) => {
     if (err) {
-      topology.close(true);
+      if (topology) topology.close();
```

Comment:
how is it possible to have a `topology` that is falsey when we had to call `topology.connect` in order to get here?

Reply:
You are right, sorry!
I haven't noticed, that "topology" was replaced by "newTopology" in the callback 4 month ago.
Now, everything works well.
Thanks!